src/test/java/POM/BasePage.java

```diff
@@ -11,11 +11,11 @@
 import java.time.Duration;
 
 public class BasePage {
-    WebDriver driver;
+   protected WebDriver driver;
     WebDriverWait wait;
     Actions actions;
```

Comment:
**I recommend: make the variables protected**, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

For example

`protected Actions actions;`